qute.ls/com.redhat.qute.ls/src/main/resources/com/redhat/qute/project/datamodel/qute-resolvers.json

```diff
@@ -25,6 +25,39 @@
 				"{#for r in recordsList.take(3)}"
 			],
 			"url": "https://quarkus.io/guides/qute-reference#collections"
+		},
+		{
+			"signature": "or(arg : T) : boolean",

```

Comment:
I think it should be:

`"signature": "or(arg : T, base : java.lang.Object) : T"`

Reply:
Please write test like https://github.com/redhat-developer/quarkus-ls/blob/master/qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/resolvers/OrEmptyValueResolverTest.java